crates/iota-genesis-builder/src/stardust/migration/migration.rs

```diff
@@ -234,7 +243,7 @@ impl Migration {
     /// Verify the ledger state represented by the objects in
     /// [`InMemoryStorage`](iota_types::in_memory_storage::InMemoryStorage).
     pub fn verify_ledger_state<'a>(
-        &self,
+        &mut self,
         outputs: impl IntoIterator<Item = &'a (OutputHeader, Output)>,
```

Comment:
Do we need mut here? Removed and compiled, no problem. Trying to understand why clippy does not detect it.

Reply:
You’re right, fixed